cmd/clusterctl/client/cluster/crd_migration.go

```diff
@@ -167,7 +168,7 @@ func (m *crdMigrator) migrateResourcesForCRD(ctx context.Context, crd *apiextens
 			obj := list.Items[i]
 
 			log.V(5).Info("Migrating", logf.UnstructuredToValues(obj)...)
-			if err := retryWithExponentialBackoff(ctx, newWriteBackoff(), func(ctx context.Context) error {
+			if err := retryWithExponentialBackoff(ctx, newCRDMigrationBackoff(), func(ctx context.Context) error {
 				return handleMigrateErr(m.Client.Update(ctx, &obj))
 			}); err != nil {
```

Comment:
I think the following doesn't make sense anymore: https://github.com/kubernetes-sigs/cluster-api/blob/d754ad5df262f1c5054c0582c29d9bf377452f04/cmd/clusterctl/client/cluster/crd_migration.go#L133-L135

Please check & update if necessary. I would assume that servedVersions can be dropped